app/assets/javascripts/editor.js

```diff
@@ -85,45 +85,50 @@ function snippet_changed(item){
   $('#association').toggle(!checked);
 }
 
-function create_editor(item) {
-  item.parent().prepend("<div id='editor1'></div>");
-  item.hide();
-  Editor = ace.edit("editor1");
+function create_editor() {
+  var editorId = "editor" + Math.floor((Math.random() * 100) + 1),
```

Comment:
According to the birthday paradox, this will fail with 50% probability on 10 editors.